### PR TITLE
Use real recommender logic (based on usage).

### PR DESCRIPTION
**Pull request by kgrygiel**

### CODE REVIEW

**Comment by kgrygiel** [comment]
cc @tkulczynski

File: vertical-pod-autoscaler/recommender/model/types.go [hunk 1]
```diff
@@ -50,6 +50,11 @@ func CoresFromCPUAmount(cpuAmunt ResourceAmount) float64 {
 	return float64(cpuAmunt) / 1000.0
 }
 
+// QuantityFromCPUAmount converts CPU cores to a resource.Quantity.
```
Comment (tkulczynski) [inline vertical-pod-autoscaler/recommender/model/types.go]: Cores? I tihnk millicores, no?

File: vertical-pod-autoscaler/examples/hamster-vpa.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: "poc.autoscaling.k8s.io/v1alpha1"
```
Comment (tkulczynski) [inline vertical-pod-autoscaler/examples/hamster-vpa.yaml]: I think hamster-deployment and hamster-vpa might be a single yaml file, unless you really want to use them separately which I doubt.

File: vertical-pod-autoscaler/recommender/model/types.go (hunk 1 above)
Comment (kgrygiel) [reply vertical-pod-autoscaler/recommender/model/types.go]: Done. Changed to "ResourceAmount". It shouldn't mention units at all.

File: vertical-pod-autoscaler/examples/hamster-vpa.yaml (hunk 2 above)
Comment (kgrygiel) [reply vertical-pod-autoscaler/examples/hamster-vpa.yaml]: Done.

**Comment by tkulczynski** [comment]
Thanks, LGTM now, handing over to @bskiba.

**Comment by bskiba** [comment]
I missed this before, but are examples/ and samples/ fundamentally different or we can merge them?

**Comment by kgrygiel** [comment]
I missed that too. I'm going to merge it. Thanks.

**Pull request merged**